pkg/receipt/receipt.go

```diff
@@ -34,3 +35,13 @@ func Store(plugin index.Plugin, dest string) error {
 	err = ioutil.WriteFile(dest, yamlBytes, 0644)
 	return errors.Wrapf(err, "write plugin receipt %q", dest)
 }
+
+// Load reads the plugin receipt at the specified destination.
+// If not found, it returns os.IsNotExist error.
+func Load(path string) (*index.Plugin, error) {
```

Comment:
This method returns a pointer instead of an object, which is inconsistent with how krew passes around plugin manifests internally. Although I favor the pointer approach, I think that consistency is worth more.
Being consistent also means, that this function can be a one-liner.

Reply:
I can't recall why I made it a pointer. I probably was thinking you making things *index.Plugin in another PR recently. There's no particular reason not to do this.